jc/parsers/route_print.py

```diff
@@ -0,0 +1,565 @@
+r"""jc - JSON Convert `route-print` command output parser
+
+
+Usage (cli):
+
+    $ route print | jc --route-print
+
+Usage (module):
+
+    import jc
+    result = jc.parse('route-print', route_print_command_output)
+
+Schema:
+
+    {
+        "interface_list": [
+            {
+                "interface_index": integer,
+                "mac_address": string,
+                "description": string
+            }
+        ],
+        "ipv4_route_table": {
+            "active_routes": [
+                {
+                    "network_destination": string,
+                    "netmask": string,
+                    "gateway": string,
+                    "interface": string,
+                    "metric": string
+                }
+            ],
+            "persistent_routes": [
+                {
+                    "network_address": string,
+                    "netmask": string,
+                    "gateway_address": string,
+                    "metric": string
+                }
+            ]
+        },
+        "ipv6_route_table": {
+            "active_routes": [
+                {
+                    "interface": integer,
+                    "metric": string,
+                    "network_destination": string,
+                    "gateway": string
+                }
+            ],
+            "persistent_routes": [
+                {
+                    "interface": integer,
+                    "metric": string,
+                    "network_destination": string,
+                    "gateway": string
+                }
+            ]
+        }
+    }
+
+    Notes:
+        - The `metric` field is typically an integer but can sometimes be set to "Default"
```

Comment:
Wonder if there is another way of handling this? Could a metric of -1 or even `null`/`None` be the "Default"? Or another field called `metric_is_default` set to `True` or something?

Reply:
When "Default" is present, the interface is using the value currently set as operating system's default metric value. I believe this value is stored somewhere in the registry.

I'm not opposed to a -1 for default. I think I'd prefer that over using null/None since I usually assume if its null/None that it was not present in the original output.

Reply:
I started looking into doing this and I found an example where a metric was already set to -1. 
`persistent_routes": [
          {
            "interface": 0,
            "metric": "4294967295",
            "network_destination": "2001:db8::/64",
            "gateway": "fe80::1"
          }
`
I only googled quickly, but it looks like it may be used for Windows to automatically optimize routing.

Leaving it as a string for now. Open to discuss further.

Reply:
Just seems a shame to have this be a string field because of that one possible value. I sorta like the idea of having it be null/None if it is default and then set the `metric_is_default` field to `True`. A bit more work, but fixes this type override issue.